Builds/CMake/CMakeFuncs.cmake

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
-  message(WARNING "In-source builds are strongly discouraged.")
+  message(WARNING "In-root builds are strongly discouraged.")
```

Comment:
Part of the problem here is that I can't find a clear, precise definition of "in-source". That's causing a bit of confusion about whether a build under the `build/` directory is technically in-source or out-of-source, and so I made up a new term! :)